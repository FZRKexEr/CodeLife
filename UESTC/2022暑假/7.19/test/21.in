10
cswgzcrarznwuulnevtuqezoqudskmzxfngjdtppvqmgsvzgztkfjlzmalxzjdiaxgzvxiblwwbzleougywzdydjpnntthkzbqrt
2
lrabhowpavkbxaesyodnovjnmayjmidmycgddaejqxjxwycelgbiufziqzghrqzihtpickgprscjbyrxrioknvtrpgomzqnjbptn
10
uzvjzziythribumzdcocdhrvjwlzicgcufvofijhknqwlfghhmfehfqsoqgixxnmvcaailsyijtiduoemiqjslsbmayufbcwvfgm
5
wavcvyykynqjeobnvdvmajjfpbqtwbtuyixoaeeaxgmpbugorkspxxtuggahevsjoztfbmlheqeetnlkdpbbrgxiahyjtcyepmcl
7
wrhjrrcjmkxqdlwkcuifarapexulkqveggdfcfhpqpvemeznolyfnppzozkznqjxaqntlrovvncehtrfsbuuawledzgihgyhxgec
3
uqmaeswkkhugrwbgivubayonnaykljuiywzknlqpgppjgqnyrzladezbosaolxkpftxibylhbghghwakzjxdaklhsgibhzcppulr
2
ebjpwvjwvbofytfkglxpxpaooetobdqnrqxffzjdtqbodatkwxqtlaqtanjmegkbnmrzyrvsuvyoxffhrojzzoccchdoaqrcmfuq
10
imnoicdoglibymqyiyudnjkkgxkjeozwsjknsgiayzvvpsjirlyvljwequcnhvjkckbzdjpdxkrjqqwpgjecvtshfeslaxzhifhb
4
qubuhmritmplqeyyzdupxzjkxoqymqtqmjwqcmbwtiivplctlllkuaprdonqunbkrgyxueokhmezazqajycrcjfhfnqidbehvwiq
8
zxrgdumfafognnxuravxmghzhxmkvqhsmdzhrsgpjvqfbjjqmbhaxgbowsfsxlwlcaroqzcgzvpuymbcsquslhwjnbwfipwjivlq
9
