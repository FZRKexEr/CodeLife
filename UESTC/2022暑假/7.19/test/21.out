65980984
1024
1048576
16384
603979768
65980984
1024
33554432
4096
2048
